src/store/modules/order/actions.ts

```diff
@@ -0,0 +1,47 @@
+import { ActionTree } from 'vuex'
+import store from '@/store'
+import RootState from '@/store/RootState'
+import OrderState from './OrderState'
+import * as types from './mutation-types'
+import router from '@/router'
+
+
+const actions: ActionTree<OrderState, RootState> = {
+  updateOrderList ({ commit }, payload) {
+    commit(types.ORDER_LIST_UPDATED,  payload );
+  },
+  modifyCsv ({ commit }, payload) {
+    commit(types.MODIFY_CSV, payload);
+  },
+  async groupProducts ({commit}, csv: any) {
+    const productIds = csv.map((item: any) => {
+      return item.shopifyproductSKU
+    })
+    const viewSize = process.env.VUE_APP_VIEW_SIZE;
```

Comment:
I think value should be productIds length

Reply:
Made the requested change.